src/hooks/screen/useChatScreenHook.ts

```diff
@@ -42,6 +56,14 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
     }
   };
 
+  const handleScrollTo = (messageId: string) => {
+    const index = chats.findIndex((message) => message?.id === messageId);
+    if (index !== -1) {
+      flatListRef.current?.scrollToIndex({index});
+      setSelectedMessageId(messageId);
+    }
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new function `handleScrollTo` is introduced to scroll to a specific message in the chat. This seems like a useful feature for user experience, but make sure to handle potential exceptions or errors that might occur during the scrolling process.
```diff
+ const handleScrollTo = (messageId: string) => {
+     const index = chats.findIndex((message) => message?.id === messageId);
+     if (index !== -1) {
+       flatListRef.current?.scrollToIndex({index});
+       setSelectedMessageId(messageId);
+     }
+ };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->